### PR TITLE
add controllers and views

### PR DESCRIPTION
**Pull request by MateeSaafi**
# Project requirements
-  **Important:** do NOT use `rails generate scaffold` command. The point of this project is to show you how to set up controllers by yourself.
-  Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes.

-  For each URL you should create:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-  Make sure that you added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```
Comment (prolajumokeoni) [inline app/controllers/posts_controller.rb]: - [ ] Kindly add posts_controller #show actions, `def show; end`  
- It's okay for the actions to be empty for now 👍🏿

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```
Comment (prolajumokeoni) [inline app/controllers/users_controller.rb]: - [ ] Kindly add users_controller  #index and #show actions.
````ruby
 
  def index; end

  def show; end
````
- It's okay for the actions to be empty for now 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @matee-safi 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Controllers,  Views and Routes setup correctly 👏  



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @matee-safi 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Created all controllers and views for the Blog app.👍
✔ Made suggested changes👍
✔ Descriptive PR👍

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**